src/components/Bookings.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
-import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import Search from "./Search";
+import SearchResults from "./SearchResults";
+import * as FakeBookings from "../data/fakeBookings.json";
```

Comment:
why you think using * as here is good?, (this is common in TS but not in JS 
Read more 
https://stackoverflow.com/questions/54585763/what-is-the-difference-between-import-as-react-from-react-vs-import-react-fr